src/services/getCharacteristicsFromFirebase.tsx

```diff
@@ -1,18 +1,87 @@
 import { db } from '../firebase';
 import { doc, getDoc } from 'firebase/firestore';
 
-export const getCharacteristicsFromFirebase = async () => {
+interface Characteristics {
+  age?: string;
+  charisma?: number;
+  class?: string;
+  description?: string;
+  dexterity?: number;
+  endurance?: number;
+  health?: number;
+  intelligence?: number;
+  level?: string;
+  name?: string;
+  notes?: string;
+  personalityTraits?: string;
+  race?: string;
+  skillsNotes?: string;
+  strength?: number;
+  weakness?: string;
+  wisdom?: number;
+  weapons?: any[];
+  spells?: any[];
+  equipment?: any[];
+  imageURL?: string;
+}
```

Comment:
To powinno być w jakimś folderze globalnym  `types` w folderze z nazwą interfejsu, to na pewno nam się przyda w wielu miejscach.

Dodatkowo, `any[]` totalnie odpada, musimy tutaj to otypować.